src/app/features/fss-search/fss-search.component.ts

```diff
@@ -71,34 +77,21 @@ export class FssSearchComponent implements OnInit {
     this.displayMessage = false;
   }
 
-  refreshToken() {
-    this.displayLoader = true;
-    this.msalService.loginPopup().subscribe(response => {
-      localStorage.setItem('claims', JSON.stringify(response.idTokenClaims));
-      const idToken = response.idToken;
-      localStorage.setItem('idToken', idToken);
-      this.msalService.instance.setActiveAccount(response.account);
-      //refreshToken endpoint call to set the cookie after user login
-      this.fileShareApiService.refreshToken().subscribe(res => {
-        this.displayLoader = false;
-        this.analyticsService.login();
-        this.handleAdvancedSearchTokenRefresh();
-      });
-    });
-    this.hideMessage();
-  }
-
   onAdvancedSearchClicked(fssAdvancedSearch: any) {
+    this.displayLoader = true;
```

Comment:
displayLoader should not be set here. Please check it.

Reply:
Updated as suggested. Thanks